docs/source/Plugin/P003_commands.repl

```diff
@@ -3,33 +3,55 @@
         :widths: 20, 30
 
         "
-        ``ResetPulseCounter,<taskIndex>``
+        ``ResetPulseCounter``
+
+        legacy method:
+ 
+        ``ResetPulseCounter,<taskNumber>``
+
         ","
         Reset the counters (pulse counter and total counter) of P003 Pulse Counter.
         
-        The taskIndex parameter is optional.  When not given, only the first task with this plugin will be cleared.
+        When using multiple instances of Pulse Counter tasks, use the the taskNumber as prefix. E.g. ``2.ResetPulseCounter`` addresses task number 2.
+        When taskNumber is neither given as prefix nor as parameter, the first active Pulse Counter task will be affected.
 
-        A call to reset the value will also trigger a call to ``PLUGIN_READ`` as if it processes a normal successful read.
+        The command will also trigger a call to ``PLUGIN_READ`` as if it processes a normal successful read.
         "
         "
-        ``SetPulseCounterTotal,value,<taskIndex>``
+        ``SetPulseCounterTotal,value``
+
+        legacy method:   
+
+        ``SetPulseCounterTotal,value,<taskNumber>``
+
         ","
         Set the value of the total count of P003 Pulse Counter.
         
-        The taskIndex parameter is optional.  When not given, only the total count of the first task with this plugin will be set.
+        When using multiple instances of Pulse Counter tasks, use the the taskNumber as prefix. E.g. ``2.SetPulseCounter,1000`` sets the value in task number 2.
+        When taskNumber is neither given as prefix nor as parameter, the first active Pulse Counter task will be affected.
```

Comment:
To make the rules more portable the taskname can also be used (that is one of the reasons this [taskname]. prefixing was added, the square brackets are optional), so it seems helpful to mention that here too.

Reply:
Right, but I only wanted to give an example and not repeat all, what is already explained in the Command Reference page.
[](https://espeasy.readthedocs.io/en/latest/Reference/Command.html#command-a-specific-task-for-multiple-instances-of-a-plugin)

Originally I wanted to use a reference "please see for more details in ...", but I did not find a way nor an example to put a functioning link from P003_commands.repl to that reference page. I found something in "_plugin_substitutions_p00x.repl" but no reference to documentation in "espeasy.readthedocs.io" and I am unsure if that is the right way to do it.

Perhaps you can tell me how to put a proper link in the "P003_commands.repl" file.

Reply:
Since I didn't get a response, I mention now also the taskname option (without a link). I think that should do now.
I'll upload the changed docu today.